modules/identity-provider-oidc-about.adoc

```diff
@@ -0,0 +1,52 @@
+// Module included in the following assemblies:
+//
+// * authentication/identity_providers/configuring-oidc-identity-provider.adoc
+
+[id="identity-provider-oidc-about_{context}"]
+= About OpenID Connect authentication
+
+The Authentication Operator in {product-title} requires that the configured OpenID Connect identity provider implements the link:https://openid.net/specs/openid-connect-discovery-1_0.html[OpenID Connect Discovery] specification.
+
+ifdef::openshift-origin[]
+You can link:https://www.keycloak.org/docs/latest/server_admin/index.html#openshift[configure a Keycloak] server as an OpenID Connect identity provider for {product-title}.
+endif::[]
+
+[NOTE]
+====
+`ID Token` and `UserInfo` decryptions are not supported.
+====
+
+By default, the `openid` scope is requested. If required, extra scopes can be specified in the `extraScopes` field.
+
+Claims are read from the JWT `id_token` returned from the OpenID identity provider and, if specified, from the JSON returned by the `UserInfo` URL.
+
+At least one claim must be configured to use as the user's identity. The standard identity claim is `sub`.
+
+You can also indicate which claims to use as the user's preferred user name, display name, and email address. If multiple claims are specified, the first one with a non-empty value is used. The following table lists the standard claims:
+
+[cols="1,2",options="header"]
+|===
+
+|Claim
+|Description
+
+|`sub`
+|Short for "subject identifier." The remote identity for the user at the
+issuer.
+
+|`preferred_username`
+|The preferred user name when provisioning a user. A shorthand name that the user wants to be referred to as, such as `janedoe`. Typically a value that corresponding to the user's login or username in the authentication system, such as username or email.
+
+|`email`
+|Email address.
+
+|`name`
+|Display name.
+|===
+
+See the link:http://openid.net/specs/openid-connect-core-1_0.html#StandardClaims[OpenID claims documentation] for more information.
+
+[NOTE]
+====
+Unless your OpenID Connect identity provider supports the resource owner password credentials (ROPC) grant flow, users must get a token from `<oauth_server_hostname>/oauth/token/request` to use with command-line tools.
```

Comment:
https://docs.openshift.com/container-platform/4.11/authentication/understanding-authentication.html calls `namespace_route` instead of `oauth_server_hostname`, need we update understanding-authentication.html with `oauth_server_hostname` for consistency?

Reply:
@xingxingxia Thanks for bringing this up. In searching, I actually see a lot of other places (44 hits) using <namespace_route> in this and other OAuth paths.

For consistency, I'll actually change mine to <namespace_route> for the time being, and then if we think <oauth_server_hostname> is better, then I can update to this in a future PR. 

@stlaz Can you weigh in on whether <namespace_route> is fine to use throughout, or if we should update all instances to <oauth_server_hostname>?

Reply:
@bergerhoffer https://docs.openshift.com/container-platform/4.11/authentication/understanding-authentication.html explains `<namespace_route> refers to the namespace route. This is found by running the following command` to users. However, in your PR's files, no such explanation for users looking at those pages, this may be a problem.